pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/ProtoWriterTools.java

```diff
@@ -1396,20 +1389,22 @@ public static int sizeOfIntegerList(FieldDefinition field, List<Integer> list) {
             return 0;
         }
         int size = 0;
+        final int listSize = list.size();
         switch (field.type()) {
             case INT32 -> {
-                for (final int i : list) {
-                    size += sizeOfVarInt32(i);
+                for(int i = 0; i < listSize; i++) {
```

Comment:
What are spotless rules for whitespaces between `for` and `(`?

Reply:
We are not running spotless on PBJ so far. I tried and the changes were huge. So backed it out. Feel we need to do another PR to get everything to spotless then maintain. If we want PBJ to match spotless.